src/popup/components/TransferReview.vue

```diff
@@ -291,14 +301,14 @@ export default defineComponent({
         }
         const transaction: IPendingTransaction = {
           hash: txResult.hash,
-          amount,
```

Comment:
You need to rework `categorizeContractCallTxObject` in order to change the structure of pending transaction.